oodeel/methods/dknn.py

```diff
@@ -110,3 +104,13 @@ def requires_to_fit_dataset(self) -> bool:
             bool: True if `fit_dataset` is required else False.
         """
         return True
+
+    @property
+    def requires_internal_features(self) -> bool:
+        """
+        Whether an OOD detector needs a `fit_dataset` argument in the fit function.
+
+        Returns:
+            bool: True if `fit_dataset` is required else False.
```

Comment:
Same as the previous comment and for all following methods. The docstring should be updated